src/server/routers/tips.ts

```diff
@@ -157,4 +160,80 @@ export const tipsRouter = router({
 
       return lesson
     }),
+  markTipComplete: baseProcedure
+    .input(
+      z.object({
+        tipId: z.number(),
+      }),
+    )
+    .mutation(async ({input, ctx}) => {
+      const token = ctx?.userToken
+
+      if (!token) return null
+      const {tipId} = input
+
+      const headers = {
+        Authorization: `Bearer ${token}`,
+        'X-SITE-CLIENT': process.env.NEXT_PUBLIC_CLIENT_ID as string,
+        'Content-Type': 'application/json',
+      }
+
+      let res = await fetch(
+        `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/watch/download`,
+        {
+          method: 'POST',
+          headers,
+          body: JSON.stringify({
+            lesson_view: {
+              lesson_id: tipId,
+            },
+          }),
+        },
+      ).then((res) => res.json())
```

Comment:
```suggestion
      // This is posting to /watch/download since it creates the proper amount of segments watched on the lesson view where /watch/manual_complete does not.
      let res = await fetch(
        `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/watch/download`,
        {
          method: 'POST',
          headers,
          body: JSON.stringify({
            lesson_view: {
              lesson_id: tipId,
            },
          }),
        },
      ).then((res) => res.json())
```

Reply:
I think leaving a comment here would be beneficial since it's an unusual situation with the API